pkg/policy/api/rule.go

```diff
@@ -241,6 +241,15 @@ type EgressRule struct {
 // Example: 192.0.2.1/32
 type CIDR string
 
+// L4Proto is a layer 4 protocol name
+type L4Proto string
+
+const (
+	ProtoTCP L4Proto = "tcp"
```

Comment:
What's the overlap between these and the L4Protocol constants defined in api/v1/models/port.go ?

Reply:
We aren't managing the policy API via swagger yet so this is separate. I would rather not depend on the models file in the policy API for now. We can discuss options on how to consolidate.